modelator/ModelResult.py

```diff
@@ -128,6 +130,9 @@ def __str__(self):
 
                 self._write_traces(s, indent, op)
 
+            for trace in self._simulation_traces:
```

Comment:
Does this print also the paths to the traces? I think `_simulation_traces` only contains the string representation.

Reply:
You are right, for consistency with other commands, I will add printing of paths (rather than printing of traces)